src/TodoList.js

```diff
@@ -1,5 +1,66 @@
-class TodoList {
+const todoApp = {
+  id: 0, // id for keeping track of each object valule
```

Comment:
Very nice, this makes the removal of items from the array easier, because we are sure the items won't ever have the same id !